### PR TITLE
enable get trust relationships for managed wallets

### PR DESCRIPTION
**Pull request by samwel141**
## Description

**Issue(s) addressed**

- Resolves #

**What kind of change(s) does this PR introduce?**

- [ ] Enhancement
- [x] Bug fix
- [ ] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

**What is the new behavior?**

## Breaking change

**Does this PR introduce a breaking change?**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "treetracker",
-  "version": "1.38.4",
+  "version": "1.38.0",
```
Comment (dadiorchen) [inline package.json:3]: We shouldn't change and push this file, changing version number is automatically down by the semantic release

File: server/app.js [hunk 2]
```diff
@@ -6,8 +6,17 @@ const { errorHandler, handlerWrapper } = require('./utils/utils');
...
+  origin: '*',
+  methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
+  credentials: true,
+  optionsSuccessStatus: 204,
+};
+
 if (process.env.NODE_ENV === 'development') {
   app.use(cors());
+} else {
+  app.use(cors(corsOptions));
```
Comment (dadiorchen) [inline server/app.js:19]: Why we need this? The api gateway can help use handle the cors problem.

**Review by dadiorchen — COMMENTED** [review]
And please check the CI/githug action, why it failed?

**Comment by samwel141** [comment]
The tests in the code are now passing, but I wonder why checks have failed again here

**Comment by samwel141** [comment]
@Kpoke may you also check the issue

**Pull request closed without merging**